pkg/kubelet/clustertrustbundle/clustertrustbundle_manager.go

```diff
@@ -74,29 +136,29 @@ func NewInformerManager(ctx context.Context, bundles certinformersv1alpha1.Clust
 	// apply to them.
 	_, err := m.ctbInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
 		AddFunc: func(obj any) {
-			ctb, ok := obj.(*certificatesv1alpha1.ClusterTrustBundle)
+			ctb, ok := obj.(*certificatesv1beta1.ClusterTrustBundle)
```

Comment:
these probably have to be parameterized somehow... hard-coding beta types in the generic doesn't seem right (which also means we didn't test this working on the alpha)